src/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./commands";
+export * from "./settings";
```

Comment:
Hopefully, this is fine to add. While in library mode - you construct the settings object manually, so having Settings types exposed explicitly would be helpful. Example:

```ts
import { Settings } from 'graphile-migrate';
// import { Settings } from 'graphile-migrate/dist/settings'; // Without this change, had to import like this
// ...
const settings: Settings = {
    logger: new Logger(),
    connectionString: config.dbOwnerConnectionString,
    shadowConnectionString: config.dbShadowConnectionString,
    rootConnectionString: config.pgRootConnectionString,
    pgSettings: {
      search_path: 'app_public,app_private,app_hidden,public',
    },
    //...
  }
```

Reply:
I'd prefer we pick the specific exports, I'm trying to move away from `export *` in new code.

```suggestion
export  { Settings } from "./settings";
```